custom_background/models/report.py

```diff
@@ -603,7 +662,7 @@ def _run_wkhtmltopdf(  # noqa: C901
         for temporary_file in temporary_files:
             try:
                 os.unlink(temporary_file)
-            except (OSError, IOError):  # noqa: disable=B014
```

Comment:
why remove?

Reply:
it's removed automatically from the pre-commit